scripts/git-secrets-check.sh

```diff
@@ -1,29 +1,55 @@
 #!/bin/bash
 set -e
+islocal=$1
 
 if [ -d /tmp/git-secrets ]; then
     echo The command git-secrets is available
 else
     echo The command git-secrets is not available, cloning...
     git clone git@github.com:awslabs/git-secrets.git /tmp/git-secrets/
     if [ -f /tmp/git-secrets/git-secrets ]; then
-	echo "Moving git secrets into PATH"
-        sudo cp /tmp/git-secrets/git-secrets /usr/sbin/
+        if [[ $(uname -s) == "Darwin" ]]; then  # Mac OSX check
+            gs_path="/usr/local/bin"
+        else # Linux, we're likely running in CircleCI
+            gs_path="/usr/sbin"
+        fi
+
+        echo "Moving git secrets into PATH"
+        sudo cp /tmp/git-secrets/git-secrets $gs_path/
+        $gs_path/git-secrets --install -f
```

Comment:
Add `rm -rf /tmp/git-secrets` and maybe change the availability check to `$gs_path` to make this more idempotent